dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/relationship/DefaultRelationshipService.java

```diff
@@ -89,8 +88,8 @@ public Relationships getRelationships(RelationshipOperationParams params)
     return Relationships.of(relationships, pager);
   }
 
-  public int countRelationships(RelationshipQueryParams queryParams) {
-
+  private int countRelationships(RelationshipQueryParams queryParams) {
+    // TODO(ivo) should we pass something other than null?
```

Comment:
@enricocolasante do you have an opinion?

Reply:
I think for now it is ok. I would worry about this when we tackle the harmonization of the stores.